src/script/ceph-release-notes

```diff
@@ -104,9 +104,9 @@ def split_component(title, gh, number):
         issue_labels = {it['name'] for it in issue['labels']}
         if 'documentation' in issue_labels:
             return 'doc: ' + title
-        item = labels.intersection(issue_labels)
+        item = (labels.union(set(prefixes))).intersection(issue_labels)
```

Comment:
What is the purpose of combining labels  with prefixes? Will `issue_labels` ever match an element of `prefixes` that is not already in `labels`?